regression/goto-analyzer/constant_propagation_07/test.desc

```diff
@@ -3,7 +3,7 @@ main.c
 --constants --simplify out.gb
 ^EXIT=0$
 ^SIGNAL=0$
-^Simplified:  assert: 1, assume: 0, goto: 3, assigns: 13, function calls: 0$
```

Comment:
This doesn't look right?  I can imagine it could alter the number of things simplified but not the number of assignments.

Reply:
Well, it just changed the number of assignments _that were simplified_ (well, "modified" perhaps would be a more accurate description as those "simplifications" were just re-arrangements of the operands).

Reply:
Ah, yes, good point.  Thanks.